src/freon.c

```diff
@@ -163,6 +167,9 @@ int ioctl(int fd, unsigned long int request, ...) {
             if (lockfd != -1) {
                 TRACE("Telling Chromium OS to regain control\n");
                 ret = FREON_DBUS_METHOD_CALL(TakeDisplayOwnership);
+                if (WEXITSTATUS(ret) == 1) {
```

Comment:
will it always be 1? or just non-zero?

Reply:
Looks like it will always exit with 1 when an error occurs: https://cgit.freedesktop.org/dbus/dbus/tree/tools/dbus-send.c#n593